### PR TITLE
Add initial migration of labware definitions from sqlite database

### LINKED ISSUE #762: Labware Files: Add New Labware Definitions to Repo
As a Robot API user (specifically: a container load function), I'd like to be able to access our default labware data, user custom labware data, and labware offsets within our repo, so that I can construct labware (for the robot, and for inclusion in protocol files). 

## Acceptance Criteria
- 'labware data', 'custom_definitions' and 'offsets' added to repo
-- added to npm package.json
-- added to docker container filesystem w/copy

### PR DESCRIPTION
**Pull request by btmorr**
## overview

Add initial migration of default labware definitions from sqlite database to new format (per RFC #746 )

Fixes #762 

## changelog

- (feature) Add labware definitions as repo-level project with package.json for direct import into JS projects

## review requests

Check that package.json is formed correctly and if anything else is needed for import into JS projects

### CODE REVIEW

File: labware-definitions/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "labware-definitions",
```
Comment (mcous) [inline labware-definitions/package.json]: We might want to scope this with our organization: `@opentrons/labware-definitions`

* Protects us from an accidental publish to npm (alternative, add `"private": true`)
    * Public publishing is still possible, but must be explicitly opted into via a CLI flag
* Generally a good idea for internal libraries (see `@opentrons/components`)

File: labware-definitions/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "version": "0.1.0",
+  "description": "Default labware definitions for Opentrons robots",
+  "main": "index.js",
```
Comment (mcous) [inline labware-definitions/package.json]: Probably want to remove this line until if/when we actually have something `require`able

File: labware-definitions/package.json (hunk 1 above)
Comment (IanLondon) [reply labware-definitions/package.json]: 👍

File: labware-definitions/package.json (hunk 2 above)
Comment (IanLondon) [reply labware-definitions/package.json]: Do you think `index.js` will import & export all the labware .json's? I can't think of a way around that, `Deck` component will need all the definitions...

Comment (mcous) [reply labware-definitions/package.json]: @IanLondon to begin, I think something like [webpack dynamic imports](https://webpack.js.org/api/module-methods/#import-) (maybe with `"webpackMode": "eager"` if we don't want to mess with code-splitting just yet) would work as a stop gap.

I think this brings up the important question of how we expect to be getting this data in production:

* Some sort of labware API?
* A labware library that's bundled into one JS with all the definitions (e.g. `index.js`, here)?
* A more nuanced library that can somehow lazy-load the definitions?
    * Could be driven by an API (1) or the filesystem (2)

**Pull request merged**